requirements.txt

```diff
@@ -9,7 +9,6 @@ python-rake>=1.0.7
 sklearn
 scipy==0.18.0
 keras==1.2.1
-tensorflow==0.12.1
+tensorflow==1.1.0
```

Comment:
I tested it. works absolutly fine with 1.1.0 as needed 1.1.0 to run keras with gpu